index.html

```diff
@@ -0,0 +1,173 @@
+             <p class="desc2">@Munyonyo Commonwealth Resort, Along Lake Victoria Shores</p>
+             <!-- <div class="header-icons"> <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+             </div> -->
+        </div>
+      
+
+                 <div class="program-wrapper">
+                  <div class="title-wrap">
+                  <h3 class="pro-header">Main Program</h3>
+                  <hr>
+                 </div>
+        
+                      <div class="main-program">
+          
+                        <div class="program-card">
+                          <img src="images/speech.png" alt="" class="pro-icon">
+                          <h4 class="program">Lecture</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/inov2.png" alt="Innovators Exhibition" class="pro-icon">
+                          <h4 class="program">Inovators</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/invest.png" alt="Investments Forum" class="pro-icon">
+                          <h4 class="program">Investers</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/net.jpg" alt="Networking" class="pro-icon">
+                          <h4 class="program">Networks</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                       </div>
+
+                              <div class="advert">
+                                <p>Join WFH Dev's Annual global  2022</p>
+                              </div>
+                              <div class="advert1">
+                                <p>SEE THE WHOLE PROGRAM</p>
+                              </div>
+                 </div>
+        <div class="speaker-head">
+        <h3 class="speaker-header">Featured Speakers</h3>
+        <hr>
+      </div>
+        <div class="speakers"></div>
```

Comment:
- [ ] In the speakers section you need to follow the [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design in terms of font, the underlines the background visual. 

**I have pointed to the elements that require changes in the speaker section of your project, in the left picture.**

Behance Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/183242307-553d2c8f-0b80-494f-80b7-2502a9b4815b.png)  |  ![image](https://user-images.githubusercontent.com/60618877/183242230-9218896d-999c-4169-b47b-eda96b4c8513.png)